### ISSUE TITLE
Install templates from git repo is broken?

### ISSUE THREAD

**Issue by somedev** [rambler-digital-solutions/Generamba#135]
I'm trying to add custom template to my `Rambafile`:

``` yaml
templates:
- {name: swift_vip, git: 'https://github.com/somedev/generamba-template-test'}
```

After running `generamba template install` I've got an error:

```
/Users/ed/.rvm/gems/ruby-2.2.4/gems/generamba-1.0.0/lib/generamba/template/installer/remote_installer.rb:33:in `install_template': Cannot find swift_vip.rambaspec in the root directory of specified repository. (StandardError)
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/generamba-1.0.0/lib/generamba/template/processor/template_declaration.rb:32:in `install'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/generamba-1.0.0/lib/generamba/template/processor/template_processor.rb:42:in `block in install_templates'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/generamba-1.0.0/lib/generamba/template/processor/template_processor.rb:40:in `each'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/generamba-1.0.0/lib/generamba/template/processor/template_processor.rb:40:in `install_templates'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/generamba-1.0.0/lib/generamba/cli/template/template_install_command.rb:18:in `install'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/thor-0.19.1/lib/thor/invocation.rb:115:in `invoke'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/thor-0.19.1/lib/thor.rb:235:in `block in subcommand'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
    from /Users/ed/.rvm/gems/ruby-2.2.4/gems/generamba-1.0.0/bin/generamba:5:in `<top (required)>'
    from /Users/ed/.rvm/gems/ruby-2.2.4/bin/generamba:23:in `load'
    from /Users/ed/.rvm/gems/ruby-2.2.4/bin/generamba:23:in `<main>'
    from /Users/ed/.rvm/gems/ruby-2.2.4/bin/ruby_executable_hooks:15:in `eval'
    from /Users/ed/.rvm/gems/ruby-2.2.4/bin/ruby_executable_hooks:15:in `<main>'
```

My template repo: https://github.com/somedev/generamba-template-test
Also I've tested with other git templates (ex. https://github.com/etolstoy/default_template) with same result

Environment:
Mac OS X 10.11.6 (15G31)
Ruby 2.2.4p230 (2015-12-16 revision 53155) [x86_64-darwin15]
Generamba 1.0.0

**Comment by etolstoy** [comment]
Hi, @somedev. We'll check it ASAP.

**Comment by etolstoy** [comment]
@somedev fixed - we'll publish the 1.1.0 release during the next week.

**Comment by somedev** [comment]
Great, thanks 👍
